dune-project

```diff
@@ -0,0 +1,5 @@
+(lang dune 2.0)
```

Comment:
Dune 2.7 maybe? Given that we're up to that now...

Reply:
Changed